src/AssimpWorker/import/AssimpImporter.cpp

```diff
@@ -23,7 +23,9 @@ namespace AssimpWorker {
 
 	const aiScene* AssimpImporter::importSceneFromFile(std::string fileName, Log& log){
 		importer.SetPropertyBool(AI_CONFIG_PP_FD_REMOVE, true); //remove degenerate polys
+		importer.SetPropertyBool(AI_CONFIG_IMPORT_NO_SKELETON_MESHES, true); //do not import skeletons
 		importer.SetPropertyInteger(AI_CONFIG_PP_SBP_REMOVE, aiPrimitiveType_LINE | aiPrimitiveType_POINT); //Drop all primitives that aren't triangles
+		importer.SetPropertyBool(AI_CONFIG_IMPORT_COLLADA_IGNORE_UP_DIRECTION, true);
```

Comment:
I'm not really happy with throwing away information from the original files here. According to the [assimp source](https://github.com/assimp/assimp/blob/master/code/ColladaLoader.cpp#L177-L191) that only affects the (virtual) root node of the scene anyway, which we can (hopefully) skip if needed.